tutorials/docker/project/build.gradle

```diff
@@ -0,0 +1,22 @@
+plugins {
+    id "com.diffplug.gradle.eclipse.mavencentral" version "3.18.0"
+}
+
+apply plugin: 'java'
+apply plugin: 'application'
+
+mainClassName = 'DemoApp'
+
+repositories {
+    maven { url = 'https://europe-maven.pkg.dev/jxbrowser/releases' }
+    mavenCentral()
+}
+
+ext {
+    jxBrowserVersion = '7.16'

```

Comment:
This tutorial project should be a child module of the root project. See the root `build.gradle` in the repo.

I see that the Gradle configuration in every tutorial is implemented differently, that's wrong. Let's quickly discuss this during the sync-up today.

Reply:
I do not think that you can use this approach in this scenario. The only files available in the Docker container are those which we add with the `ADD` command. Because of this, this project should be self-sufficient and do not rely on other files.